NEWS.md

```diff
@@ -1,3 +1,22 @@
+## 4.8.9 2021-09-19
```

Comment:
```suggestion
## 4.8.3 2021-09-19
```

Reply:
That was a mistake.. I'm brewing on Okapi 4.9.0 here.. https://github.com/folio-org/okapi/tree/OKAPI-1032-release-okapi-4-9-0 and will fix the typo here.